packages/ui/resolution/tests/ResolutionDecorator-specs.js

```diff
@@ -1,20 +1,17 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
 import ResolutionDecorator from '../ResolutionDecorator';
 
 describe('ResolutionDecorator Specs', () => {
-
 	test('should apply resolution classes to the wrapped component', () => {
 		const Component = ResolutionDecorator('div');
-		const subject = mount(
-			<Component />
-		);
-
-		const div = subject.find('div');
+		render(<Component data-testid="component" />);
 
-		const expected = true;
-		const actual = (div.hasClass('enact-res-standard') && (div.hasClass('enact-orientation-landscape') || div.hasClass('enact-orientation-portrait')));
+		const div = screen.getByTestId('component');
 
-		expect(actual).toBe(expected);
+		expect(div).toHaveClass('enact-res-standard');
+		expect(div).toHaveClass('enact-orientation-portrait');
```

Comment:
I think it's not the same with (div.hasClass('enact-orientation-landscape') || div.hasClass('enact-orientation-portrait').